addons/mail/tests/test_mail_render.py

```diff
@@ -101,6 +108,11 @@ def setUpClass(cls):
     <span>Autre Narrateur</span>
 </p>"""
         ]
+        cls.base_rendered_local_links = [
```

Comment:
Actually this could be defined directly in ``test_render_template_local_links`` test (no need to bloat cls with data used only once :) )